Dockerfile-alpine.template

```diff
@@ -0,0 +1,28 @@
+# A GeoServer installation
+FROM tomcat:%%TOMCAT_VERSION%%-jre%%JRE_VERSION%%-alpine
+
+ENV GEOSERVER_VERSION  %%GEOSERVER_VERSION%%
+ENV GEOSERVER_WAR_URL  https://github.com/azavea/docker-geoserver/releases/download/0.0.0/geoserver-${GEOSERVER_VERSION}-war.zip
+ENV GEOSERVER_MONITORING_URL https://github.com/azavea/docker-geoserver/releases/download/0.0.0/geoserver-${GEOSERVER_VERSION}-monitor-plugin.zip
+ENV GEOSERVER_DIR  ${CATALINA_HOME}/webapps/geoserver
+ENV GEOSERVER_EXT_DIR  ${GEOSERVER_DIR}/WEB-INF/lib
+ENV GEOSERVER_DATA_DIR /data
+
+RUN cd /tmp \
+  && rm -rf ${CATALINA_HOME}/webapps/* \
+  && mkdir -p ${GEOSERVER_DATA_DIR} ${GEOSERVER_DIR}\
+  && wget -O /tmp/geoserver.zip ${GEOSERVER_WAR_URL} \
```

Comment:
I think that it would be valuable to silence the progress output from `wget` here. It generates quite a bit of noise in the build logs.